eng/Version.Details.xml

```diff
@@ -380,6 +380,10 @@
       <Uri>https://github.com/aspnet/Extensions</Uri>
       <Sha>321f30556d63ec8af6eb0bc43d534b6b2e5458a0</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.DotNet.GenAPI" Version="1.0.0-beta.19119.2">
+      <Uri>https://github.com/dotnet/arcade</Uri>
+      <Sha>000000</Sha>
```

Comment:
Run `darc update-dependencies --source-repo https://github.com/dotnet/arcade` to get the latest SHA. (Note: I may have missed a required command line arg.)

Reply:
This is done according to our guidance: https://github.com/aspnet/AspNetCore/blob/master/docs/ReferenceResolution.md

Reply:
I need to clarify that guidance.  000000 is a suggestion, not an actual rule

> If you don't know the commit hash of the source code used to produce "0.0.1-beta-1", you can use 000000 as a placeholder

If you do know the commit hash, you might as well add it.